Makefile

```diff
@@ -442,6 +443,16 @@ bundle: kustomize operator-sdk manifests set-image-controller
 	$(OPERATOR_SDK) bundle validate ./bundle
 	./hack/ignore-createdAt-bundle.sh
 
+.PHONY: reset
+reset: kustomize operator-sdk manifests
+	$(MAKE) VERSION=${OPERATOR_VERSION} set-image-controller
+	$(OPERATOR_SDK)  generate kustomize manifests -q
+	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle -q --overwrite --version ${OPERATOR_VERSION} $(BUNDLE_METADATA_OPTS)
+	$(OPERATOR_SDK) bundle validate ./bundle
+	./hack/ignore-createdAt-bundle.sh
+	git checkout config/manager/kustomization.yaml
+	OPERATOR_IMG=local/opentelemetry-operator:e2e TARGETALLOCATOR_IMG=local/opentelemetry-operator-targetallocator:e2e OPERATOROPAMPBRIDGE_IMG=local/opentelemetry-operator-opamp-bridge:e2e DEFAULT_OPERATOR_IMG=$(IMG) DEFAULT_TARGETALLOCATOR_IMG=$(TARGETALLOCATOR_IMG) DEFAULT_OPERATOROPAMPBRIDGE_IMG=$(OPERATOROPAMPBRIDGE_IMG) SED_BIN="$(SED)" ./hack/modify-test-images.sh
```

Comment:
Wouldn't it be enough to just `git restore tests`?

Reply:
Nevermind, that doesn't work if you've actually made changes to the E2E tests. :unamused: